docs/components/truncate.md

```diff
@@ -0,0 +1,230 @@
+
+When this prop is set tu true, component applies different logic - truncation is achieved through `-webkit-line-clamp` CSS property, rather than using height of child elements to determine the row height.
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate is-text-content :rows="3">
+      <p>A good design decision to apply text truncation would be when displaying a <b>large amount of textual content</b>, such as a list of articles or product descriptions, in a limited space, such as a mobile screen or a small widget. By truncating the text to a short summary, it is possible to present the information in a more organized and readable manner, allowing the user to quickly scan and understand the main points of each item.</p>
+      <p>The truncated text can also serve as a teaser, encouraging the user to click or tap to view the full content.</p>
+    </KTruncate>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KTruncate is-text-content :rows="3">
+    <p>A good design decision to apply text truncation would be 
+      when displaying a <b>large amount of textual content</b>, such as a list 
+      of articles or product descriptions, in a limited space, such as 
+      a mobile screen or a small widget. By truncating the text to a 
+      short summary, it is possible to present the information in a more 
+      organized and readable manner, allowing the user to quickly scan 
+      and understand the main points of each item.</p>
+      <p>The truncated text can also serve as a teaser, encouraging the user to click or tap to view the full content.</p>
+  </KTruncate>
+</template>
+```
+
+### isExpanded
+
+If passed `true`, the component will be rendered expanded by default.
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate is-expanded>
+      <KBadge v-for="n in 25" :key="n">
+        Item {{ n }}
+      </KBadge>
+    </KTruncate>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KTruncate is-expanded>
+    <KBadge v-for="n in 25" :key="n">
+      Item {{ n }}
+    </KBadge>
+  </KTruncate>
+</template>
+```
+
+## Slots
+
+### default
+
+Slot for content.
+
+To ensure the best experience with the component, please pass elements of equal height in the default slot. The component will base its guess for the height of the element on the height of each individual sibling, so passing elements of equal height will result in a more accurate and consistent output.
+
+:::tip TIP
+The component is reactive to it's dimensions changes. To see it in effect, try resizing your browser window and see the behavior of the example below.
+:::
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate :rows="3">
+      <KBadge v-for="n in 30" :key="n">
+        Item {{ n }}
+      </KBadge>
+    </KTruncate>
+  </template>
+</KCard>
+
+### expand-trigger
+
+Slot for custom expand trigger element. Slot props:
+- `excessElementsCount` (type: `Number`) - Number of elements that overflow. **Note**: this slot prop is only available when not `isTextContent`
```

Comment:
question: instead of `excessElementsCount` would something like `truncatedCount` be easier/make more sense?

Reply:
Good idea, I like that name better